adapters/httpapi_compact.c

```diff
@@ -1273,7 +1286,7 @@ static bool validRequestType(HTTPAPI_REQUEST_TYPE requestType)
 
 HTTPAPI_RESULT HTTPAPI_ExecuteRequest_Internal(HTTP_HANDLE handle, HTTPAPI_REQUEST_TYPE requestType, const char* relativePath,
     HTTP_HEADERS_HANDLE httpHeadersHandle, const unsigned char* content,
-    size_t contentLength, unsigned int* statusCode, HTTP_HEADERS_HANDLE responseHeadersHandle,
+    size_t contentLength, unsigned int* statusCode, char* reasonPhrase, HTTP_HEADERS_HANDLE responseHeadersHandle,
```

Comment:
need add one more parameter telling length of the buffer, otherwise there could be buffer over run